garage-deploy/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM ubuntu:18.04
+
+ENV UBUNTU_VERSION 18.04
+ENV GARAGE_DEPLOY_VERSION 2018.11
+
+RUN  apt-get update \
+  && apt-get install -y wget \
+  && wget https://github.com/advancedtelematic/aktualizr/releases/download/$GARAGE_DEPLOY_VERSION/garage_deploy-ubuntu_$UBUNTU_VERSION.deb \
```

Comment:
could maybe check the checksum here, similar to this https://github.com/advancedtelematic/infra-dockerfiles/blob/master/network-tools/Dockerfile#L20

But it's pointed at our github release so it ain't so pressing an issue.